### ISSUE TITLE
WSL unexpectedly uses dumb terminal

### ISSUE THREAD

**Issue by hyee** [jline/jline3#451]
With `v3.9`, the JLine terminal for WSL on Win10 is `PosixSysTerminal`, after upgrading to `v3.13` it becomes a dumb terminal with message `WARNING: Unable to create a system terminal, creating a dumb terminal (enable debug logging for more information)`,  the logger prints following exceptions:
```
java.lang.NumberFormatException: For input string: ""
        at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
        at java.lang.Integer.parseInt(Integer.java:592)
        at org.jline.utils.InfoCmp.parseInfoCmp(InfoCmp.java:582)
        at org.jline.terminal.impl.AbstractTerminal.parseInfoCmp(AbstractTerminal.java:195)
        at org.jline.terminal.impl.PosixSysTerminal.<init>(PosixSysTerminal.java:44)
        at org.jline.terminal.TerminalBuilder.doBuild(TerminalBuilder.java:366)
        at org.jline.terminal.TerminalBuilder.build(TerminalBuilder.java:259)
```

Similar exception happens when creating terminal with `jna(true)` instead of `jansi(true)`

Sample code:
```java
Terminal t=TerminalBuilder.builder().system(true).jna(false).jansi(true).signalHandler(Terminal.SignalHandler.SIG_IGN).nativeSignals(true).build();
```

**Comment by gnodet** [comment]
Yeah, this looks like a regression caused by the fix for #408

**Comment by gnodet** [comment]
I'll cut a new release now.
